docs/en/02_Developer_Guides/03_Forms/Field_types/02_DateField.md

```diff
@@ -51,19 +51,16 @@ strtotime()).
 
 	:::php
 	DateField::create('MyDate')
-		->setConfig('min', '-7 days')
-		->setConfig('max', '2012-12-31')
+		->setMinDate('-7 days')
+		->setMaxDate'2012-12-31')
 		
 ## Separate Day / Month / Year Fields
 
-The following setting will display your DateField as three input fields for day, month and year separately. HTML5 
-placeholders 'day', 'month' and 'year' are enabled by default. 
+To display separate input fields for day, month and year separately you can use the `DateFieldSeparated` subclass`.
```

Comment:
This references `DateFieldSeparated` rather than `SeparatedDateField` - I've fixed this in a follow-up commit.